src/index.js

```diff
@@ -60,22 +60,17 @@ export default function ({ types: t }) {
         }
       },
       MemberExpression(path, { opts = {} }) {
-
         let { node } = path;
 
-        opts.optimize = (opts.optimize === true) || false;
```

Comment:
This runs multiple times for every MemberExpression, so I don't think it makes sense to re-initialize it every time. Also, do you care if it's false, null or undefined?